src/tscpp/api/CMakeLists.txt

```diff
@@ -45,5 +45,10 @@ add_library(ts::tscppapi ALIAS tscppapi)
 target_link_libraries(tscppapi
     PUBLIC
     libswoc
-    yaml-cpp::yaml-cpp)
+    yaml-cpp::yaml-cpp
+    )
 install(TARGETS tscppapi)
+
+if (APPLE)
+    set_target_properties(tscppapi PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
```

Comment:
```suggestion
    # does this need to be PUBLIC I wonder?
    target_link_options(tscppapi PUBLIC "-undefined dynamic_lookup")
```

Reply:
Fixed

Reply:
It could be PUBLIC, but the plugins still need to be dynamic_lookup because they're plugins.